<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilteredSearchWidgetPage</class>
 <widget class="QWidget" name="FilteredSearchWidgetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>716</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTextEdit" name="exampleText">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="emptyLabel">
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="emptyFilteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="withTextLabel">
         <property name="text">
          <string>With text</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="filteredSearchWidgetWithText" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
         <property name="textFilter" stdset="0">
          <string>Lemon</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="withFiltersLabel">
         <property name="text">
          <string>With filters</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="filteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="withEnabledFiltersLabel">
         <property name="text">
          <string>With enabled filters</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="enabledFilteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="shortLabel">
         <property name="text">
          <string>Short without filters</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="shortUnfilteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="shortWithFiltersLabel">
         <property name="text">
          <string>Short with filters</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="shortFilteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="shortWithEnabledFiltersLabel">
         <property name="text">
          <string>Short with enabled filters</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="AzQtComponents::FilteredSearchWidget" name="shortEnabledFilteredSearchWidget" native="true">
         <property name="placeholderText" stdset="0">
          <string>Use a keyword or regular expressions</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AzQtComponents::FilteredSearchWidget</class>
   <extends>QWidget</extends>
   <header>AzQtComponents/Components/FilteredSearchWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
